.gitlab-ci.yml

```diff
@@ -1405,69 +1414,79 @@ kitchen_windows_upgrade6-a7:
 # run dd-agent-testing on centos
 kitchen_centos_chef-a6:
   allow_failure: false
+  needs: ["deploy_rpm_testing-a6"]
```

Comment:
This is done in the `kitchen_centos_common` phase, and I think will overwrite the other needs that are listed there.

Reply:
I have removed it from `kitchen_centos_common`, because we had both `deploy_rpm_testing-a6 ` and `deploy_rpm_testing-a7`, so both agent kitchen tests had to wait for both agent builds.